config/database.yml

```
@@ -84,6 +84,7 @@ production:
     database: <%= ENV['LAGO_CLICKHOUSE_DATABASE'] %>
     host: <%= ENV['LAGO_CLICKHOUSE_HOST'] %>
     port: <%= ENV.fetch('LAGO_CLICKHOUSE_PORT', 8123) %>
+    pool: <%= ENV.fetch('DATABASE_POOL', 10) %>
```

I'm wondering if we necessarily want the same value for PG and ClickHouse 🤔 

How do you feel about `LAGO_CLICKHOUSE_POOL`, I know we already have a lot of of env vars 🙈 

```suggestion
    pool: <%= ENV.fetch('LAGO_CLICKHOUSE_POOL', 10) %>
```

---

we use clickhouse in HTTP, so it's not really a POOL, it's just for AR to be happy when we have a lot of requests, adding one more env var here is useless I think